hy-mode.el

```diff
@@ -267,8 +267,9 @@ commands."
               #'run-jedhy--pyvenv-post-deactive-hook)))
 
 (defun hy-mode--support-company ()
-  "Support `company-mode' autocompletion."
-  (add-to-list 'company-backends #'company-hy))
+  ;; "Support `company-mode' autocompletion."
```

Comment:
Why comment-out the documentation string?

Reply:
eldoc trys to read it, chokes, and causes errors. I think this may be related to some changes in emacs-28.

Reply:
That sounds like a local Emacs problem (if only because that docstring is perfectly valid Elisp), and&mdash;of course&mdash;local problems should not affect code changes in a public project like this.

Reply:
Yep. Removed the change since it should not have been in in the first place.